sensu-plugin.gemspec

```diff
@@ -20,5 +20,6 @@ Gem::Specification.new do |s|
   s.add_dependency('mixlib-cli', '>= 1.5.0')
 
   s.add_development_dependency('rake')
+  s.add_development_dependency('rubocop')
```

Comment:
we should lock it to a pessimistic patch version or just about every other PR this is gonna change based on what people already have installed.